client_test.go

```diff
@@ -70,6 +70,10 @@ func TestClientGet(t *testing.T) {
 	_, err = client.Get("/url")
 	assert.NoError(t, err)
 
+	// Attribute "response" can be non-array - it doesn't cause pagination failure.
+	gock.New(testURL).Get("/url").Reply(200).BodyString(`{"response":"a string"}`)
+	assert.NoError(t, err)
```

Comment:
We are missing the `client.Get()` call here, though I think we can add the Body mock to the previous test at line #69 and then add this check after line #71:
```
assert.Equal(t, "a string", res.Get("response").String())
```

Reply:
Oh, what, terribly sorry - fixed it as you suggested.